languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -56241,6 +56241,87 @@ though, and has since been slightly modified)-->
+                <pattern>
+                    <marker>
+                        <token regexp="yes">conformes?</token>
+                        <token>avec</token>
+                    </marker>
+                </pattern>
+                <message>'\1 \2' est une expression fautive. </message>
+                <suggestion>conforme à</suggestion>
+                <suggestion>en conformité avec</suggestion>
+                <example correction="conforme à|en conformité avec">Ce produit est <marker>conforme avec</marker> sa description.</example>
+                <example>Ce produit est conforme à sa description.</example>
+            </rule>
+            <rule id="CONFORME_2" name="conformité a->avec">
+                <pattern>
+                    <marker>
+                        <token>en</token>
+                        <token regexp="yes">conformités?</token>
+                        <token>à</token>
+                    </marker>
+                </pattern>
+                <message>'\1 \2 \3' est une expression fautive. </message>
+                <suggestion>conforme à</suggestion>
+                <suggestion>en conformité avec</suggestion>
+                <example correction="conforme à|en conformité avec">Ce produit est <marker>en conformité à</marker> la legislation.</example>
+                <example>Ce produit en conformité avec la legislation.</example>
+            </rule>
+            <rule id="CEST_CE_SONT" name="c'est->ce sont">
+                <antipattern>
+                    <token>c'</token>
+                    <token skip="2">est</token>
+                    <token postag_regexp="yes" postag="R (pers|dem) .*"/>
+                </antipattern>
+                <pattern>
+                    <marker>
+                        <token>c'</token>
+                        <token skip="2">est</token>
+                    </marker>
+                    <token postag_regexp="yes" postag=".* p|sp .*"/>
+                </pattern>
+                <message>'C'est' suivi d'un pluriel devient 'ce sont'. Dans la langue parlée, on admet 'c'est' suivi par 'ceux/celles' ou 'eux|elles'</message>
+                <suggestion>ce sont</suggestion>
+                <example correction="Ce sont"><marker>C'est</marker> des barbares qui ont fait ça.</example>
+                <example>C'est eux qui ont eu l'idée.</example>
+                <example>C'est ceux qui ont eu l'idée.</example>
+            </rule>
+            <rule id="SE_BAFRER" name="se bâfrer">
+                <pattern>
+                    <marker>
+                        <token postag_regexp="yes" postag="R pers obj .*"/>
+                        <token postag_regexp="yes" postag="V etre .*" skip="1" min="0"/>
+                        <token postag_regexp="yes" postag="V .*" inflected="yes">bâfrer</token>
+                    </marker>
+                </pattern>
+                <message>'Bâfrer' ne prend pas de forme pronominale</message>
```

Comment:
The message is missing a period (full stop) in the end.

Reply:
done